src/utils/contractHelpers.ts

```diff
@@ -50,15 +50,25 @@ import easterNftAbi from 'config/abi/easterNft.json'
 import cakeVaultAbi from 'config/abi/cakeVault.json'
 import predictionsAbi from 'config/abi/predictions.json'
 import chainlinkOracleAbi from 'config/abi/chainlinkOracle.json'
-import { DEFAULT_GAS_PRICE } from 'config'
+import { DEFAULT_GAS_PRICE, TESTNET_CHAIN_ID } from 'config'
 import { getSettings, getGasPriceInWei } from './settings'
 
+export const getGasPrice = (account?: string) => {
+  const chainId = process.env.REACT_APP_CHAIN_ID
+  if (chainId === TESTNET_CHAIN_ID) {
+    return 10
+  }
+  if (account) {
+    return getSettings(account).gasPrice
+  }
```

Comment:
I think you should keep this function pure (insofar as the account has nothing to do with the "config" of gas price). Adding the account context locks it into the front end app.

Reply:
Okay, maybe not "locks" but you get the idea.

Reply:
Yea sure I think I get you, I've changed the function to be explicitly getting chain defaults, although these will only be used in the absence of user settings